dapp/src/sagas/network.js

```diff
@@ -51,14 +64,13 @@ function * deduceBridgeSides (networkType) {
   }
 }
 
-function * getNetworkType ({ enableProvider }) {
+function * getNetworkType ({ enableProvider, provider }) {
```

Comment:
What you think about renaming that function? It's doing much more.
I think we can call it `connectToNetwork({ provider, connectAccount })`